package/MDAnalysis/coordinates/PDB.py

```diff
@@ -651,7 +700,25 @@ def _write_pdb_header(self):
             self.REMARK(*remarks)
         except AttributeError:
             pass
-        self.CRYST1(self.convert_dimensions_to_unitcell(u.trajectory.ts))
+
+        if u.dimensions is None or np.allclose(u.dimensions, np.zeros(6)):
+            # Unitary unit cell by default (PDB standard)
+            self.CRYST1(np.array([1.0, 1.0, 1.0, 90.0, 90.0, 90.0]))
+
+            # Add CRYST1 REMARK (285)
+            self.REMARK("285 UNITARY VALUES FOR THE UNIT CELL AUTOMATICALLY SET")
+            self.REMARK("285 BY MDANALYSIS PDBWRITER BECAUSE UNIT CELL INFORMATION")
+            self.REMARK("285 WAS MISSING.")
+            self.REMARK("285 PROTEIN DATA BANK CONVENTIONS REQUIRE THAT")
+            self.REMARK("285 CRYST1 AND SCALE RECORDS BE INCLUDED, BUT THE VALUES ON")
```

Comment:
Just state that CRYST1 records be included and don't mention SCALE. Then add a _comment_ to the code saying that we decided against including SCALE records anywhere as we are only implementing a subset of the PDB standard. (We do deal with CRYST1 so we adhere to any standard with regards to that record.)